src/stravalib/tests/integration/test_client.py

```diff
@@ -272,15 +272,18 @@ def test_get_activity_streams_series_type_unofficial(mock_strava_api, client):
         ({}, {}, None, None),
```

Comment:
@jsamoocha i do need help here. i'm a bit lost with respect to how the mocks are setup to interface as a mock API. 
i also was confused because the DeprecationWarning here being tested is not a part of our validation helper. Right now if  a user ONLY provides a activity_type to create an activity, it will throw a value error that sport type is missing. but for update activity we don't have a check. 

i guess i'm feeling confused about how to best implement this for both create and update activity. AND then i'm confused around how the tests are setup for these two methods. 

the validation helper tests do run now. Many thanks for any help that you can provide. i've only done a tiny bit of mocking so it's hard to rewrite this test as i don't fully understand what's happening.